### ISSUE TITLE
silx compare - typo

### ISSUE THREAD

**Issue by payno** [silx-kit/silx#3942]
Looks like there is a small typo on a set definition. Get the following error:

```
TypeError: set expected at most 1 argument, got 2
ERROR:silx.app.compare.CompareImagesWindow:Error while loading URL bambou_hercules_0001_rec_2058.800_00000.hdf5?/entry0000/reconstruction/results/data
Traceback (most recent call last):
  File "/home/lesaint/envs/tomotools/lib/python3.9/site-packages/silx/app/compare/CompareImagesWindow.py", line 116, in _updateImageB
    data = self.readData(urlPath)
  File "/home/lesaint/envs/tomotools/lib/python3.9/site-packages/silx/app/compare/CompareImagesWindow.py", line 141, in readData
    if data.ndim == 3 and data.shape[0] in set(3, 4):
TypeError: set expected at most 1 argument, got 2
```

**Comment by lesaintjerome** [comment]
OK. I probably need to be more precise on the issue. The error happened when with `silx compare "bambou_hercules_0001_rec_2040.800_00000.hdf5?path=/entry0000/reconstruction/results/data" "bambou_hercules_0001_rec_2058.800_00000.hdf5?path=/entry0000/reconstruction/results/data"`, i.e. with no slice number. But when the command `silx compare "bambou_hercules_0001_rec_2040.800_00000.hdf5?path=/entry0000/reconstruction/results/data&slice=0" "bambou_hercules_0001_rec_2058.800_00000.hdf5?path=/entry0000/reconstruction/results/data&slice=0"`is used (i.e. with slice number), then it works fine. That's why I thought stacks were possible....

**Comment by lesaintjerome** [comment]
It would be nice to have the possibility to compare stacks (with a slider to run through the slices, simultaneously in both stacks). Might be an enhancement request?

**Comment by vallsv** [comment]
BTW i don't think stacks are supported

**Comment by vallsv** [comment]
Closed by #3943
